qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+Feature: 1. Sign in to Quantori Scoring Portal with Google account feature
+
+ Background:
+ 	Given Quantori Scoring Portal page (QSPP) exists
+ 	And QSPP is linked to the Internet address https://qa.ddso-spot.quantori.com/
+ 	And Two versions of QSPP could be displayed by the same address: QSPP 'User NOT signed in' and QSPP 'User signed in'
+ 	And On the QSPP 'User NOT signed in' version following components are displayed:
+ 		| Component										|
+ 		| 'Sign in with Google' button					|
+ 		| Quantori Scoring	logotype image				|
+ 	And On the QSPP 'User signed in' version following components are displayed:
+ 		| Component                                     |
+ 		| Quantori Scoring	logotype image              |
+		| Divider                                       |
+ 		| 'Dataset' text line                           |
+ 		| Divider                                       |
+ 		| Drag & drop form								|
+ 		| Username text field							|
+ 		| User profile icon								|
+ 		| Sign out button with sign out icon			|
+
+ Scenario: 1_01. 'Sign in with Google' button on the QSPP 'User NOT signed in' version is interactive and hyperlinked
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	When User drags cursor to the 'Sign in with Google' button
```

Comment:
User can have several pages (e.g. app under test and some online shop  with  "sign in with Google" button) open in browser , which page is user on? Can this info be put in Background?

Reply:
I think, I would add 'current' to where browser tab user is at is mentioned, would that be ok with you?

Reply:
or you can simply write 'User is on QSPP 'User NOT signed in' version (page)', what do you think?

Reply:
Yes, thank's, that's much better, I'll change it this way.